CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ set(CLIENT_VERSION_MINOR 99)
 set(CLIENT_VERSION_BUILD 0)
 set(CLIENT_VERSION_RC 0)
 set(CLIENT_VERSION_IS_RELEASE "false")
-set(COPYRIGHT_YEAR "2025")
+string(TIMESTAMP COPYRIGHT_YEAR "%Y")
```

Comment:
nit: Add `UTC` option to avoid using local time?

Reply:
Added the `UTC` option.